src/main/java/step5/domain/Car.java

```diff
@@ -0,0 +1,34 @@
+package step5.domain;
+
+public class Car {
+
+    private static final int DEFAULT_POSITION = 0;
+
+    private final Engine engine;
+    private final String name;
+    private int position;
```

Comment:
```suggestion
    private final CarStatus carStatus;
    
    ...
    ...
    
    public static Car of(Engine engine, String name) {
        return new Car(engine, CarStatus.of(name, 0));
    }

    public CarStatus carStatus() {
        return carStatus;
    }
```

`CarStatus`를 만들어 주었으니 이런 관점으로 인스턴스 변수 수를 줄여볼수 있겠네요! 
`getter`도 줄일수 있을것 같구용 🤔